### ISSUE TITLE
Can't save fins on pods with fin count=1

### ISSUE THREAD

**Issue by neilweinstock** [openrocket/openrocket#431]
This is on Daniel's build, still labeled alpha 5.
Using this design:
[alc_new_pods.zip](https://github.com/openrocket/openrocket/files/2177152/alc_new_pods.zip)
On the pods, there are two fins: "fin tip" and "pylon".  When opening the file, each of those will open with a fin count of "3", despite the fact that I save the file with fin count of "1".

**Comment by neilweinstock** [comment]
Works correctly on the file that was previously failing in Alpha 8.

**Comment by teyrana** [comment]
Sorry, can you rephrase that? alpha8 should have fixed it :/

**Comment by neilweinstock** [comment]
Sorry, I knew the wording sounded funny when I wrote it.


It seems to work now.  I tried it on the file that was previously breaking, and it reads back correctly with one fin in each set.

________________________________

**Comment by teyrana** [comment]
Understood.  Thanks for getting back to us :)

**Comment by neilweinstock** [comment]
This problem has re-appeared for me in alpha 8.  See attached ORK file.  The trapezoidal fins on the pods should have 1 per set, but they always come back as 3 when I reload the file.
[faddle.zip](https://github.com/openrocket/openrocket/files/2298305/faddle.zip)

**Comment by neilweinstock** [comment]
I still see this problem in Alpha 9.

**Comment by neilweinstock** [comment]
I'm re-opening this one.  In the alpha I'm using now, pods with instance count = 1 are again not being saved.  Or at least, when re-opening the file they all show up with instance count = 2.

**Comment by teyrana** [comment]
@neilweinstock 

This should be fixed in unstable, now.   Are you stil seeing this?

**Comment by neilweinstock** [comment]
Nope, seems to be fixed.

**Comment by neilweinstock** [comment]
Time to close it down again, hopefully for good this time. :)
